packages/locale/src/translations/en.json

```diff
@@ -514,6 +514,9 @@
             "multishipping_guest_intro": "To ship your items to multiple addresses you need to",
             "multishipping_guest_sign_in": "sign in to your account",
             "multishipping_guest_create": "or <a href=\"{url}\" target=\"_blank\">create an account</a> prior to proceeding.",
+            "multishipping_item_to_allocate_message": "1 item left to allocate",
+            "multishipping_item_to_allocate_message_plural": "{itemCount} items left to allocate",
```

Comment:
Hey @bc-mackxu ,
You can use https://github.com/bigcommerce/checkout-js/blob/7950f36cc392989229d87e8417c686c3b3f94c82/packages/locale/src/translations/en.json#L67 this way for singular and plural strings.